### DISCUSSION TITLE
How marks coordinates work?

### DISCUSSION THREAD

**Discussion by arkadiuszwojcik** [cetz-package/cetz#556] [General]
I am confused with marks coordinates. I am using 0.2.2 and doc is saying that first parameter is place position, and second is direction, so I test following code:

```
  mark((0,0), (0,1), symbol: "stealth", scale: 1, fill: black)
  mark((0,0), (0,3), symbol: ">", scale: 1, fill: black)
```

I would expect both marks to be in same place (0,0) and point in same direction. But somehow second parameter effect their position and they end up in different places:

![Screenshot 2024-04-14 010846](https://github.com/cetz-package/cetz/assets/2368976/e086687d-b508-4a55-a7ab-84f19dc52a04)

Moreover I would like to write function that places mark at the center of line, but because of that bug? It doesn't work. Here is my function:

```
let draw-arrow-line(start, end) = {
  line(start, end)
  mark((start, 0.5, end), end, symbol: "stealth", scale: 1, fill: black)
}
```

Can someone advice what I am doing wrong?

**Comment by fenjalien** [comment]
Turns out this is a bug, #557 was opened to resolve this.
